tests/unit/helpers/format-message-(html)-test.js

```diff
@@ -25,14 +25,23 @@ module('format-message (html)', function (hooks) {
   });
 
   test('arguments marked as safe-string is not escaped', function (assert) {
-    assert.expect(1);
+    assert.expect(2);
+
     assert.equal(
       this.intl.formatMessage(`'<strong>'Welcome {name}!'</strong>'`, {
         htmlSafe: true,
         name: htmlSafe('<em>Alexander</em>'),
       }).string,
       '<strong>Welcome <em>Alexander</em>!</strong>'
     );
+
+    assert.equal(
+      this.intl.formatMessage(`<strong>Welcome {name}!</strong>`, {
+        htmlSafe: true,
+        name: htmlSafe('<em>Alexander</em>'),
+      }).string,
+      '<strong>Welcome <em>Alexander</em>!</strong>'
+    );
```

Comment:
Should we for all these also assert for `isHTMLSafe(str)` or is this sufficiently covered in other tests already?

Reply:
I would probably rather use `String(...)` or `.toString()` over `.string`, but I don't see this as an issue.

https://github.com/emberjs/ember.js/blob/9ad9b7b2ff80e1100f6b278171412f4b558b111b/packages/%40ember/-internals/glimmer/lib/utils/string.ts#L5-L19

I also made it a habit (and use a lint rule for it in my projects) to always use `assert.strictEqual()` over `assert.equal()`, but again, not important.